lib/processor.js

```diff
@@ -5,6 +5,9 @@ const uglify = require('uglify-es');
 const getConfig = require('./config');
 const config = getConfig();
 
+const MAX_LINE_COUNT_OF_SHORT_EXAMPLES = 10;
```

Comment:
I think perhaps we should set this a bit lower. It seems nice to have a consistent size for the examples even where they don't quite fill the editor, and therefore to try to make both "smaller" and "taller" fairly exceptional.

Back when we introduced these custom sizes I did a quick analysis of the size distribution: https://github.com/mdn/interactive-examples/issues/464#issuecomment-360572522 - this is a bit out of date but still probably close to accurate. It tells us that most JS examples are 8-12 lines long. And I think an 8 line example looks fine in the standard editor height.

So if we set this to `7`, we will have pretty good consistency, so we use the standard height for most examples. But at the same time we will cater for the very short examples, like https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getDate, which look silly in a standard-height editor.

What do you think?

 I think if we set this to 7, we will

Reply:
I agree it is better to use the standard height for as many examples as we can.